pkg/infra/config_test.go

```diff
@@ -68,5 +68,63 @@ client_per_conn: 40`
 				ClientPerConn: 40,
 			}))
 		})
+
+		It("Load node without TLS", func() {
```

Comment:
I'm not sure how much value this case adds... it just verifies that yaml can be loaded, which is `tls` irrelevant.